benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-sandl-nonprod.json

```diff
@@ -2,5 +2,6 @@
     {"LiveFrom": "10/06/2022", "CaseTypeID": "Benefit", "CaseEventID": "listingError", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
     {"LiveFrom": "10/06/2022", "CaseTypeID": "Benefit", "CaseEventID": "listingError", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
     {"LiveFrom": "01/06/2022", "CaseTypeID": "Benefit", "CaseEventID": "updateListingRequirements", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
-    {"LiveFrom": "01/06/2022", "CaseTypeID": "Benefit", "CaseEventID": "updateListingRequirements", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"}
+    {"LiveFrom": "01/06/2022", "CaseTypeID": "Benefit", "CaseEventID": "updateListingRequirements", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
+    {"LiveFrom": "04/07/2022", "CaseTypeID": "Benefit", "CaseEventID": "addCaseOutcome", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"}
```

Comment:
Should we have more roles for all these? Currently on a super user would be able to perform the event and use the fields

Reply:
Authorisation for caseworker & superuser added.

Reply:
You'll need to add the same for the event

Reply:
Does this need doing?